docs/docs/04-standard-library/06-ex/dynamodb-table.md

```diff
@@ -1,14 +1,10 @@
 ---
-title: API reference
-id: api-reference
-description: Wing standard library API reference for the ex module
-keywords: [Wing sdk, sdk, Wing API Reference]
-hide_title: true
-sidebar_position: 100
+title: Dynamodb Table
```

Comment:
This seems like an unintended change, right?

Reply:
<img width="716" alt="image" src="https://github.com/winglang/wing/assets/39455181/375b2dbe-ebb2-42b3-a64c-bb3d091d06d6">
github is just being confused - it thinks I took the api-reference file and renamed it into "dynamodb-table" while I deleted the api-reference and added a new dynamodb md file